diff --git a/dist/commonjs/index.js b/dist/commonjs/index.js
index be9df86be09c7655787a65c55ae6da01858894c3..dff24f80c4c640f978e2cd34d5afd183bdbd551e 100644
--- a/dist/commonjs/index.js
+++ b/dist/commonjs/index.js
@@ -260,4 +260,8 @@ function expand_(str, max, maxLength, isTop) {
     }
     return acc;
 }
+module.exports = expand;
+module.exports.expand = expand;
+module.exports.EXPANSION_MAX = exports.EXPANSION_MAX;
+module.exports.EXPANSION_MAX_LENGTH = exports.EXPANSION_MAX_LENGTH;
 //# sourceMappingURL=index.js.map
